pkg/util/globalerror/errors.go

```diff
@@ -61,16 +61,16 @@ const (
 
 	StoreConsistencyCheckFailed ID = "store-consistency-check-failed"
 	BucketIndexTooOld           ID = "bucket-index-too-old"
+
+	MsgSizeTooLarge ID = "msg-size-too-large"
 )
 
 // Message returns the provided msg, appending the error id.
 func (id ID) Message(msg string) string {
 	return fmt.Sprintf("%s (%s%s)", msg, errPrefix, id)
 }
 
-// MessageWithLimitConfig returns the provided msg, appending the error id and a suggestion on
-// which configuration flag(s) to use to change the limit.
-func (id ID) MessageWithLimitConfig(msg, flag string, addFlags ...string) string {
+func (id ID) MessageWithLimitConfigBuilder(template, msg, flag string, addFlags ...string) string {
```

Comment:
Could you rebase `main`, please? A similar change has been merged and now this string is built into the new function `buildFlagsList()`.